src/e2e/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
```

Comment:
I feel like something like this file could be useful in `main` and `test` too...?

Reply:
sure, I'll add, and let's see if any benefit! 👍🏻